index.bs

```diff
@@ -1245,7 +1245,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 ::  whose value is the bytes of the signature value returned by the [=authenticator=].
 
                 :   <code><dfn for="assertionCreationData">userHandleResult</dfn></code>
-                ::  whose value is the bytes of the [=user handle=] returned by the [=authenticator=].
+                ::  If the authenticator returned a [=user handle=], set the value of [=userHandleResult=] to be the bytes of that
+                    [=user handle=] returned by the [=authenticator=]. Otherwise, set the value of [=userHandleResult=] to null.
```

Comment:
suggest:
If the [=authenticator=] returned a [=user handle=], set the value of [=userHandleResult=] to be the bytes of the returned [=user handle=]. Otherwise, ...

Reply:
Thanks!